### ISSUE TITLE
data/20180525.pkl  not found error

### ISSUE THREAD

**Issue by akshayudnur** [YueLiu/NeuralTripleTranslation#4]
data/20180525.pkl file not available in data folder

**Comment by YueLiu** [comment]
hi @akshayudnur, there were two tf generated files you got after training the model. because they were too large so I could not upload them. you should be able to get them once you ran the end-end training. I suggest you try to use seq2seq/s2s_preprocess.py to generate the .pkl file. there s a local path needed for pretrained word2vec. Then you may try baselines/main.py and see if your input dimension is valid. (you might need to swap the loss.data[0] to loss.item() in main.py)

**Comment by YueLiu** [comment]
and of course, you need to prepare your training corpus following [this pattern](https://drive.google.com/file/d/1XmtOarcCuMUGyIh851yFx5Nkp6iihdak/view?usp=sharing)
